client-v2/src/components/Editions/ManageEdition.tsx

```diff
@@ -0,0 +1,216 @@
+                  <LinkButton
+                    size="l"
+                    href={`editions/${this.state.currentIssue.id}`}
+                  >
+                    Open
+                  </LinkButton>
+                </>
+              ) : (
+                <>
+                  <InformationMsg status="info">
+                    No issue found for this date
+                  </InformationMsg>
+                  <ButtonDefault size="l" onClick={this.createEdition}>
+                    Create
+                  </ButtonDefault>
+                </>
+              )}
+            </>
+          )}
+        </div>
+      </>
+    );
+  }
+
+  private createEdition = () => {
+    if (!this.state.date) {
+      Raven.captureMessage(
+        `Creating an issue failed because no date was selected`
+      );
+      this.setState({ infoMessage: 'The date is not valid', isError: true });
+      return;
+    }
+    this.setState({ infoMessage: 'New issue created' });
+    this.createIssue(this.state.date)
+      .then(issue => this.setState({ currentIssue: issue }))
+      .catch(err => {
+        Raven.captureMessage(
+          `Creating an issue for this date ${this.state.date} failed`,
+          err
+        );
+        this.setState({
+          infoMessage: 'Sorry creating an issue for this date failed',
+          isError: true
+        });
+      });
+  };
+
+  private handleDateChange = (date: Moment | null) => {
+    this.setState({ date, infoMessage: '', isError: false });
+    if (!date) {
+      return;
+    }
+    const issue = this.selectIssueForDate(date);
+    if (issue) {
+      this.setState({ currentIssue: issue });
+    } else {
+      this.setState({ currentIssue: null });
+    }
+  };
+
+  private handleFocusChange = (isFocussedObj: { focused: boolean | null }) => {
+    if (isFocussedObj.focused) {
+      const startDate = moment().startOf('month');
+      const endDate = moment()
+        .add(1, 'month')
+        .endOf('month');
+      this.fetchIssuesForDateRange(startDate, endDate).then(issues =>
+        this.setState({ issues })
+      );
+    }
+    this.setState({ isDatePickerOpen: !!isFocussedObj.focused });
+  };
+
+  private selectIssueForDate = (date: Moment) =>
+    this.state.issues.find(i =>
+      moment(i.publishDate, 'YYYY-MM-DD HH:mm:ss-ZZ').isSame(date, 'day')
+    );
+
+  private handleMonthClick = (month: Moment) => {
+    const startDate = month.clone().startOf('month');
+    const endDate = month.clone().endOf('month');
+
+    this.fetchIssuesForDateRange(startDate, endDate)
+      .then(issues => this.setState({ issues }))
+      .catch(err => {
+        Raven.captureMessage(
+          `Fetching issues for this date range failed: ${startDate} to ${endDate}`,
+          err
+        );
+        this.setState({
+          infoMessage: `Fetching issues for this date range failed: ${startDate} to ${endDate}`
+        });
+      });
+  };
+
+  private fetchIssuesForDateRange = async (
```

Comment:
We are probably not going to return anything sensible for this for the first pass as it isn't strictly necessary to create editions but we can leave this here for now as we will definitely need it in the future